R/ab_scenario.R

```diff
@@ -77,10 +77,10 @@ ab_scenario = function(houses,
     # supressing messages associated with GEOS ops on unprojected data
     suppressWarnings({
       suppressMessages({
-        origins = houses %>% dplyr::sample_n(size = pop, replace = TRUE)
-        destination_zone = zones %>% dplyr::filter(geo_code == desire_lines$geo_code2[i])
+        origins = houses[sample(1:nrow(houses), size = pop, replace = TRUE), ]
```

Comment:
Neat solution. You can make it even more concise though:

```
        origins = houses[sample(nrow(houses), size = pop, replace = TRUE), ]
```

(No need for the `1:` bit.)